README.md

```diff
@@ -124,18 +124,16 @@ This CLI component of this package is intended to be used with NPM scripts. It f
 ## Contributing
 
 ### Releases
-This information is for project maintainers:
-
-Requirements
 
-- Have access to [`npmjs`](https://www.npmjs.com/) Shopify organization
-- Yarn must be installed
+This information is for project maintainers:
 
 Instructions
 
-- Bump the version in [`package.json`](https://github.com/Shopify/node-themekit/blob/08d421f67adf1e5c61a6bd600e40c3e0f3924216/package.json#L3)
-- Create a release on GitHub as well as tagging the release
-- Publish the package on [`npmjs`](https://www.npmjs.com/)
+- Bump the version in [`package.json`](https://github.com/Shopify/node-themekit/blob/master/package.json)
+- Run `npm install` to update `package-lock.json`
+- Commit and push the changes.
```

Comment:
Could condense these three steps by running [`npm version`](https://docs.npmjs.com/cli/version) and pushing.

Reply:
Thanks, I didn't know about that. Fixed in 27723d5

Reply:
(btw I only just noticed you're at Shopify any more, so the feedback is extra appreciated :grin:)